src/api/Alert/types.ts

```diff
@@ -38,6 +38,13 @@ export interface IAssignCasesParams extends IDataSourceParams {
   userId: number;
 }
 
+export type ICasesPerAlertAssignmentParam = Pick<IAssignCasesParams, 'alertId' | 'caseIds'>;
```

Comment:
**Just a comment, this is not a change request**

Clever way to avoid duplicating code!

An alternative could be:

```typescript
export interface ICasesPerAlertAssignmentParam {
  alertId: number;
  caseIds: number[];
}

export interface IAssignCasesParams extends ICasesPerAlertAssignmentParam, IDataSourceParams {
  userId: number;
}

export interface IBulkAssignCasesParams extends IDataSourceParams {
  casesPerAlert: ICasesPerAlertAssignmentParam[];
  assignedUserId: number;
}
```